cmd/dep/ensure.go

```diff
@@ -225,7 +225,10 @@ func (cmd *ensureCommand) Run(ctx *dep.Ctx, args []string) error {
 	// vendor does not exist we should write vendor
 	var writeV bool
 	path := filepath.Join(sw.Root, "vendor")
-	vendorIsDir, _ := dep.IsDir(path)
+	vendorIsDir, err := dep.IsDir(path)
```

Comment:
This is redundant with the IsEmptyDir check at checks to see if it's empty and a dir.